tests/api.v2/model/experiment.test.js

```diff
@@ -14,7 +14,7 @@ describe('model/experiment', () => {
         'id',
         'name',
         'description',
-        'processing_config',
```

Comment:
where did `processing_config` go?

Reply:
I had removed it because the UI doesn't need the processing config in getExperiment (it isn't returned in the v1 implementation)

Reply:
the UI loads the basic experiment's data and sets processing with a loading flag so that the processing config is loaded.

I believe this is because the GET endpoint for a single experiment is used in `getExperimentInfo` in the UI, this actions is called on the server side of the UI and I guess it was made to avoid loading too much stuff from the server side, just the essentials instead.